server/routes/creatingLicences/handlers/caseload.test.ts

```diff
@@ -39,6 +37,8 @@ describe('Route Handlers - Create Licence - Caseload', () => {
             id: 1,
             type: LicenceType.AP,
             status: LicenceStatus.IN_PROGRESS,
+            hardStopDate: addDays(parseIsoDate('2022-10-12'), -2),
```

Comment:
Not that it particularly matters, just a query as to using `addDays` with a negative value as opposed to `subDays`? Is there a reason that `subDays` isn't suitable, or just habit? 😁

Reply:
I only found subDays halfway through and didn't go back to fix 😂
Can replace in a follow up?

Reply:
I thought that might be the case but wanted to make sure there wasn't a known issue with `subDays` that I wasn't aware of 😂 
No bother, if you remember to fix it then sure, but I'm pretty certain I've seen `addDays` with negative values elsewhere so don't stress it!

Reply:
I've moved these over to use subDays now